Kickstarter-iOS/Views/Controllers/ManagePledgePaymentMethodView.swift

```diff
@@ -0,0 +1,147 @@
+  private lazy var cardLabelsStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var expirationDateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var lastFourLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+
+  private let viewModel = CreditCardCellViewModel()
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.configureViews()
+    self.setupConstraints()
+    self.bindViewModel()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Configuration
+
+  public func configure(with card: GraphUserCreditCard.CreditCard) {
+    self.viewModel.inputs.configureWith(creditCard: card)
+  }
+
+  private func configureViews() {
+    _ = self
+      |> \.accessibilityElements .~ self.subviews
+
+    _ = ([self.lastFourLabel, self.expirationDateLabel], self.cardLabelsStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.cardImageView, self.cardLabelsStackView], self.adaptableStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.titleLabel, self.adaptableStackView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = (self.rootStackView, self)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.adaptableStackView
+      |> checkoutAdaptableStackViewStyle(
+        self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+      )
+      |> adaptableStackViewStyle
+
+    _ = self.cardImageView
+      |> cardImageViewStyle
+
+    _ = self.cardLabelsStackView
+      |> cardLabelsStackViewStyle
+
+    _ = self.expirationDateLabel
+      |> cardExpirationDateLabelStyle
+
+    _ = self.lastFourLabel
+      |> cardLastFourLabelStyle
+
+    _ = self.rootStackView
+      |> checkoutCardStackViewStyle
+
+    _ = self.titleLabel
+      |> titleLabelStyle
+  }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.expirationDateLabel.rac.text = self.viewModel.outputs.expirationDateText
+    self.lastFourLabel.rac.text = self.viewModel.outputs.cardNumberTextShortStyle
+    self.viewModel.outputs.cardImage
+      .observeForUI()
+      .observeValues { [weak self] image in
+        _ = self?.cardImageView
+          ?|> \.image .~ image
+      }
+  }
+
+  // MARK: - Functions
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.cardImageView.widthAnchor.constraint(
+        equalToConstant: CheckoutConstants.PaymentSource.ImageView.width
+      )
+    ])
+  }
+}
+
+// MARK: - Styles
+
+private let adaptableStackViewStyle: StackViewStyle = { stackView in
```

Comment:
In this case I would a be a bit more specific and name it `paymentMethodAdaptableStackViewStyle` since it's super easy to confuse reader when scanning the code (made me think you were simply using the global `adaptableStackViewStyle` (without the parameter).